.github/workflows/documentation.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 1
+      max-parallel: 20
```

Comment:
does this do anything - reading the docs here: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow

I would expect this to only help if we had a matrix of jobs to run - multiple could be run at once... but we only have one documentation build i think...